packages/lodestar-cli/src/network/enr.ts

```diff
@@ -13,10 +12,6 @@ export async function writeEnr(filename: string, enr: ENR, peerId: PeerId): Prom
   await writeFile(filename, enr.encodeTxt(keypair.privateKey));
 }
 
-export async function readEnr(filename: string): Promise<ENR> {
-  return ENR.decodeTxt(await readFile(filename));
-}
-
 export async function initEnr(filename: string, peerId: PeerId): Promise<void> {
```

Comment:
This `initEnr` will write the ENR to the file differently than how FileENR does.
The difference between:
`"enr:....."` - a valid json file
and
`enr:.....` - not a valid json file, plain text file

We should standardize how the enr is written across the board to _always_ either write quoted enr or _always_ write non-quoted enr.

Imo we don't need valid json here, we can just write the enr as plain text:

so `writeEnr` should be deleted, and `initEnr` should create and save an enr using `FileENR.initFromENR(...).saveToFile();`
Also, we would change the `beaconPaths.enrFile` defaults to be `$path/enr` rather than `$path/enr.json` since we won't be writing valid json anymore.